File: abner.py
workers = int(input("Jumlah pekerja: "))
houses = int(input("Jumlah rumah: "))

bat_per_jam_per_orang = 12
total_bat_per_jam = workers * bat_per_jam_per_orang
total_bat = houses * 50

jam = total_bat / total_bat_per_jam
menit = jam * 60

print("Waktu:", int(jam), "jam", int(menit % 60), "menit")

File: abner.js
let workers = parseInt(prompt("Jumlah pekerja:"));
let houses = parseInt(prompt("Jumlah rumah:"));

let totalBat = houses * 50;
let totalPerJam = workers * 12;

let jam = totalBat / totalPerJam;
let menit = jam * 60;

console.log(`Waktu: ${Math.floor(jam)} jam ${Math.floor(menit % 60)} menit`);